src/javascript/events.py

```diff
@@ -163,7 +164,7 @@ def loop(self):
             self.threads = [x for x in self.threads if x[2].is_alive()]
 
             if len(self.freeable) > 40:
-                self.queue_payload({"r": r, "action": "free", "ffid": "", "args": self.freeable})
+                self.queue_payload({"r": 0, "action": "free", "ffid": "", "args": self.freeable})
```

Comment:
About this change: currently this does not work at all because `r` is not defined. As this calls JS without listening to a return, the request id is ignored so passing `"r": 0` works.